### PR TITLE
MNT: Add issue-forms with YAML instead of Markdown

### LINKED ISSUE #218: MNT: Add issue-forms with YAML instead of Markdown-templates
The issue-forms, as they are already implemented with [mne-python](https://github.com/mne-tools/mne-python/issues/new?assignees=&labels=BUG&projects=&template=bug_report.yml) offer a better user experience and a more streamlined issue output.

[How To Issue-Template GitHub](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository)
[Syntax Issue Forms](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema)

### PR DESCRIPTION
**Pull request by masterismail**
#### Reference issue

Fixes #218 

#### What does this implement/fix?

Added issue-forms with YAML instead of Markdown-templates for : 
bug report
feature request

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -1,20 +1,48 @@
----
-name: Feature request
-about: Suggest an idea for this project
-title: ''
-labels: ''
-assignees: ''
+name: 🛠️ Feature request
```
Comment (larsoner) [inline .github/ISSUE_TEMPLATE/feature_request.md:1]: This appears to still be a `.md` file, I think you need a `git mv`

Comment (masterismail) [reply .github/ISSUE_TEMPLATE/feature_request.md]: Yeah ! apologies for the oversight. I've corrected it now.

**Comment by larsoner** [comment]
Can you look at the pre-commit.ci failure?

**Comment by masterismail** [comment]
> Can you look at the pre-commit.ci failure?

I've solved the errors, please have a look !

**Pull request merged**